frontend/components/WatchMyPixelForm.vue

```diff
@@ -119,7 +110,21 @@ const dateValue = computed({
   set: (val) => val
 });
 
-const submitForm = handleSubmit((values) => {
+const submitForm = handleSubmit(async (values) => {
   console.log(values);
+  await useApi("/watch_my_pixel", {
+    headers: {
```

Comment:
We have to create separate method for sending authenticated requests. You can do it here or create separate issue